chart/values.yaml

```
@@ -177,3 +177,25 @@ clusterRole:
   # The name of a cluster role to bind to; if not set and create is
   # true, a name based on fullname is generated
   name: "gatekeeper-policy-manager-crd-view"
+# Health and liveness probe configurations
+livenessProbe:
+  enabled: true
+  httpGet:
+    path: /health
+    port: 8080
+  initialDelaySeconds: 10
+  periodSeconds: 10
+  timeoutSeconds: 1
+  successThreshold: 1
+  failureThreshold: 3
+
+readinessProbe:
+  enabled: true
+  httpGet:
+    path: /ready
```

we don't have a /ready endpoint. In the plain manifests we are using `/health` for both probes

```suggestion
    path: /health
```